c_src/eleveldb.cc

```diff
@@ -459,6 +465,45 @@ ERL_NIF_TERM parse_open_option(ErlNifEnv* env, ERL_NIF_TERM item, leveldb::Optio
                 opts.cache_object_warming = false;
         }
 
+        else if (option[0] == eleveldb::ATOM_EXPIRY_ENABLED)
+        {
+            if (option[1] == eleveldb::ATOM_TRUE)
+            {
+                if (NULL==opts.expiry_module.get())
```

Comment:
No.  Keep in mind that we do not know what order options will arrive.  And that cuttlefish often passes default values to common options.

I am intentionally not creating the object unless absolutely necessary.  It is highly likely that the default options will get passed ... and defaults currently leave expiry disabled.  So only creating the object if non-defaults seen.  However, I do update the object if a different non-default option caused its creation.